backend/src/engine/history-query-engine.ts

```diff
@@ -69,28 +66,18 @@ export default class HistoryQueryEngine extends BaseEngine {
       this.historyQueries.delete(settings.id);
       this.historyQueries.set(settings.id, historyQuery);
     }
-    this.historyQueries
-      .get(settings.id)!
-      .start()
-      .catch(error => {
-        this.logger.error(error);
-      });
+    await this.startHistoryQuery(settings);
   }
 
-  async addItemToHistoryQuery(historyId: string, item: SouthConnectorItemDTO): Promise<void> {
-    await this.historyQueries.get(historyId)?.addItem(item);
-  }
-
-  async deleteItemFromHistoryQuery(historyId: string, item: SouthConnectorItemDTO): Promise<void> {
-    await this.historyQueries.get(historyId)?.deleteItem(item);
-  }
-
-  async deleteAllItemsFromHistoryQuery(historyId: string): Promise<void> {
-    await this.historyQueries.get(historyId)?.deleteItems();
-  }
-
-  async updateItemInHistoryQuery(historyId: string, item: SouthConnectorItemDTO): Promise<void> {
-    await this.historyQueries.get(historyId)?.updateItem(item);
+  async startHistoryQuery(historySettings: HistoryQueryDTO): Promise<void> {
```

Comment:
Just to have the same code logic as for the oibus engine, in the creation and starting of things, it is enough to pass the `historyId` to the start function. This way, `create` needs settings, and `start` needs only an id, just like how the connectors do it:
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/engine/oibus-engine.ts#L134
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/engine/oibus-engine.ts#L149

Reply:
I just noticed that it makes the reload service code easier as well:
https://github.com/OptimistikSAS/OIBus/blob/650c83a3f23ccea1ca7d363525d373d6dc6a2a42/backend/src/service/reload.service.ts#L368-L373

Here for example, there is no need to call `const settings = this.repositoryService.historyQueryRepository.getHistoryQuery(historyId)!;` anymore, we can simply pass `historyId` to the start method.